seed-element.html

```diff
@@ -1,97 +1,110 @@
 
-      /**
-       * The `sayHello` method will return a greeting.
-       *
-       * @method sayHello
-       * @param {String} greeting Pass in a specific greeting.
-       * @return {String} Returns a string greeting.
-       */
-      sayHello: function(greeting) {
-        var response = greeting || 'Hello World!';
-        return 'seed-element says, ' + response;
-      },
+    is: 'seed-element',
 
+    published: {
       /**
-       * The `seed-element-lasers-success` event is fired whenever we
-       * call fireLasers.
-       *
-       * @event seed-element-lasers-success
-       * @param {Object} detail
-       *   @param {string} detail.sound An amazing sound.
+       * The `author` attribute sets an initial author.
        */
+      author: String,
 
       /**
-       * The `fireLasers` method will fire the lasers. At least
-       * it will dispatch an event that claims lasers were fired :)
-       *
-       * @method fireLasers
+       * `fancy` indicates that the element should don a monocle and tophat,
+       * while checking its pocket watch.
        */
-      fireLasers: function() {
-        this.fire('seed-element-lasers-success', { sound: 'Pew pew pew!' });
-      }
-
-    });
-
-  </script>
-
-</polymer-element>
+      fancy: Boolean,
+    },
+
+    // Element Lifecycle
+
+    configure: function() {
+      // `configure` is the hook to set up any initial state for the element. It
+      // is called top-down: this element's children are NOT yet initialized.
+      // See https://github.com/Polymer/polymer/blob/0.8-preview/PRIMER.md#configure-method
+      //
+      // Return your element's default values here.
+      return {
+        author: 'Dimitri Glazkov',
+      };
+    },
+
+    ready: function() {
+      // `ready` is called after all elements have been configured, but
+      // propagates bottom-up. This element's children are ready, but parents
+      // are not.
+      //
+      // This is the point where you should make modifications to the DOM (when
+      // necessary), or kick off any processes the element wants to perform.
+    },
+
+    // Element Behavior
+
+    /**
+     * The `seed-element-lasers` event is fired whenever `fireLasers` is called.
+     *
+     * @event seed-element-lasers
+     * @detail {{sound: String}}
+     */
+
+    /**
+     * Sometimes it's just nice to say hi.
+     *
+     * @param {string} greeting A positive greeting.
+     * @return {string} The full greeting.
+     */
+    sayHello: function(greeting) {
+      var response = greeting || 'Hello World!';
+      return 'seed-element says, ' + response;
+    },
+
+    /**
+     * Attempts to destroy this element's enemies with an any beam of light!
+     *
+     * Or, at least, dispatches an event in the vain hope that someone else will
+     * do the zapping.
+     */
+    fireLasers: function() {
+      var detail = {sound: 'Pew pew!'};
+      this.dispatchEvent(new CustomEvent('seed-element-lasers', {detail: detail}));
```

Comment:
Oh! Awesome. I had looked for it in the docs/source, but failed to find it.  Thanks for the pointer